packages/errors/src/errors/limit-exceeded.ts

```diff
@@ -1,3 +1,14 @@
 import { createError, ErrorCode } from '../index.js';
 
-export const LimitExceededError = createError(ErrorCode.LimitExceeded, 'Limit exceeded.', 403);
+export interface LimitExceededErrorExtensions {
+	message: string;
+}
+
+export const messageConstructor = (extensions?: LimitExceededErrorExtensions) =>
+	`${extensions?.message ? extensions.message : 'Limit exceeded.'}`;
+
+export const LimitExceededError = createError<LimitExceededErrorExtensions>(
+	ErrorCode.LimitExceeded,
+	messageConstructor,
+	403,
+);
```

Comment:
We may want to allow only the “limit category” to be passed, instead of the entire message. 
This would be more akin to the existing errors and would make it a bit easier to parse for clients.

```jsonc
{
	"message": "Active Admin users limit exceeded.",
	"extensions": {
		"code": "LIMIT_EXCEEDED",
		"message": "Active Admin users limit exceeded."
	}
}

// vs

{
	"message": "Active Admin users limit exceeded.",
	"extensions": {
		"code": "LIMIT_EXCEEDED",
		"category": "Active Admin users"
	}
}
```

```suggestion
export interface LimitExceededErrorExtensions {
	category: string;
}

export const messageConstructor = ({ category }: LimitExceededErrorExtensions) => `${category} limit exceeded.`;

export const LimitExceededError = createError<LimitExceededErrorExtensions>(
	ErrorCode.LimitExceeded,
	messageConstructor,
	403,
);
```

For the sake of completeness, we may want to add a test:

`limit-exceeded.test.ts`
```ts
import { randomAlpha, randomInteger } from '@directus/random';
import { beforeEach, expect, test } from 'vitest';
import { messageConstructor, type LimitExceededErrorExtensions } from './limit-exceeded.js';

let sample: LimitExceededErrorExtensions;

beforeEach(() => {
	sample = {
		category: randomAlpha(randomInteger(2, 500)),
	};
});

test('Constructs message', () => {
	expect(messageConstructor(sample)).toBe(`${sample.category} limit exceeded.`);
});
```

Reply:
Seems good for now :D we can always expand it to the full message if ever needed